tox.ini

```diff
@@ -17,14 +18,14 @@ deps =
     django19: {[django]1.9.x}
     django110: {[django]1.10.x}
     django_master: {[django]master}
+    py26,py27,py32: mock
     py32: coverage==3.7.1
     pytest<3.0.0
     pytest-django<3.0.0
     pytest-cov
     django-money-rates
-commands = make test
+commands = py.test --ds=tests.settings --cov=./djmoney tests
```

Comment:
Are you saying that invoking `py.test` directly from tox is 20% faster than the Make -> setup.py -> py.test chain? :O